commerce-test-util/src/main/java/com/liferay/commerce/test/util/TestCommerceContext.java

```diff
@@ -86,7 +98,11 @@ public int getCommerceSiteType() {
 
 	@Override
 	public long getSiteGroupId() throws PortalException {
-		return _contextGroup.getGroupId();
+		if (_commerceChannel == null)
+
+			return _contextGroup.getGroupId();
```

Comment:
This `if` statement should use curly brackets -- we always use brackets at Liferay, even for one-liners.